src/home/HomePage.scss

```diff
@@ -0,0 +1 @@
+@import './components/home-banner/HomeBanner';
```

Comment:
[nit]: I think we have some problems with Stylelint (this [file](https://github.com/raccoongang/frontend-app-catalog/blob/rg-tick/develop/src/index.scss) has double quotes, this file (HomePage.scss) has single quotes)